test/qpy_compat/get_versions.py

```diff
@@ -70,7 +70,7 @@ def available_versions_for_package(package, min_version=None, max_version=None):
             if not any(
                 tag in supported_tags
                 for release in payload
-                if release["packagetype"] == "bdist_wheel"
+                if release["packagetype"] == "bdist_wheel" and not release["yanked"]
```

Comment:
Just a question: how can a release be marked as "yanked"?

Reply:
Yanking has to be done manually by the project owner on pypi. It's exposed as an option in the pypi project management web ui, or via the api. The details on it are described here: https://peps.python.org/pep-0592/#motivation

Reply:
For qiskit the 1.2.3 release is the only release we've yanked so far because of the obvious typo/bug this pr fixes, which was introduced in the sole pr that made up 1.2.3.

Reply:
Ah cool, I didn't know about this -- thanks 🙂